docs/UPGRADE.md

```diff
@@ -0,0 +1,20 @@
+# HOW TO UPGRADE FORUM.SOMENERGIA.COOP
+
+## Upgrade notes
+
+Update your Gemfile:
+```
+gem "decidim", "0.XX.X"
+gem "decidim-dev", "0.XX.X"
+```
+
+Run these commands to upgrade and make sure you get all the latest migrations:
+```
+bundle update decidim
+bin/rails decidim:upgrade
```

Comment:
The same I recommended for participa.somenergia.coop here, better explain what should be executed locally and what in the server.

Reply:
Thanks